src/src/camp/camp.service.ts

```diff
@@ -2244,6 +2266,22 @@ export class CampService {
 			if (!updatedResult.success)
 				result.data.unsuccessfulReassignmentIds.push(body?.learner_id);
 
+			if (
+				old_camp_details_repsonse?.data?.group_users?.[0].group
+					?.group_users_aggregate?.aggregate?.count == 0
+			) {
+				await this.campcoreservice.updateCampStatus(
```

Comment:
change `updateCampStatus` to `updateCampGroup`

Reply:
resolved

Reply:
resolved